showtime-xyz/showtime-frontend
# fix(SHOW2-953): modal screen export

# Why

https://linear.app/showtime/issue/SHOW2-953/modalscreen-on-android

# How

extracted the modal component from the index file into a separate file.

# Test Plan

- Run yarn install
- Make sure that the version installed from Bit matches the fix
- launch the login screen
